addons/handhelds/anprc148jem/CfgWeapons.hpp

```diff
@@ -1,7 +1,7 @@
 class TFAR_anprc148jem: ItemRadio {
     author = "Raspu";
     displayName = "AN/PRC-148 JEM";
-    descriptionShort = "AN/PRC-148 JEM personal radio 5km";//#Stringtable
+    descriptionShort = "$STR_TFAR_Veh_ANPRC148JEM_Desc";
```

Comment:
to get it to CBA/ACE standards this should be STR_TFAR_Handhelds_ANP....

Reply:
Ok.

Reply:
I will look into stringtable again and fix these string into ACE3/CBA standard.